modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -84,47 +122,6 @@ export default function EditNotificationTemplate({
 		MultiSelectItem[]
 	>([]);
 
-	const validate = (values: NotificationTemplate) => {
-		const errors: NotificationTemplateError = {};
-
-		if (!values.name) {
-			errors.name = constantsUtils.REQUIRED_MSG;
-		}
-
-		if (!values.subject[defaultLanguageId]) {
-			errors.subject = Liferay.Language.get('required');
-		}
-
-		if (notificationTemplateType === 'email' || values.type === 'email') {
```

Comment:
Hi @danvb112, notice that I removed the `notificationTemplateType === 'email'` validation. Could you check to make sure there aren't any side effects that I can't see?

Reply:
To carry out further tests and ensure that this modification will not have unwanted results, we should ideally correct the current [blocker](https://liferay.atlassian.net/browse/LPD-29276). And carry out more tests with these modifications.